pan-domain-node/src/panda.ts

```diff
@@ -108,7 +110,7 @@ export class PanDomainAuthentication {
     getPublicKey(): Promise<string> {
         return this.publicKey.then(({ key, lastUpdated }) => {
             const now = new Date();
-            const diff = now.getMilliseconds() - lastUpdated.getMilliseconds();
+            const diff = now.getTime() - lastUpdated.getTime();
```

Comment:
Looks like this is a fix for separate problem? Could we add some tests for this?